CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+Contribute to DRAKVUF Sandbox
+=============================
+
+Development system
+------------------
+
+Very first thing to consider is to setup and configure your local instance of DRAKVUF Sandbox. There are two basic options in that matter:
+
+* Develop on local machine: Install Debian Buster in [VMware Workstation 15 Player](https://www.vmware.com/products/workstation-player/workstation-player-evaluation.html).
+* Develop on a remote server: Just get some bare-metal or rent a dedicated server (e.g. [Kimsufi](https://www.kimsufi.com/us/en/servers.xml)) with Debian Buster.
+
+**Caution!** Your host machine must be an Intel processor with VT-x and EPT support, even when using VMware or other nested virtualization.
```

Comment:
Maybe some oneliner to check if the processor is ok?

I guess something like

```
cpuinfo | grep xyzvwy
```

Reply:
This is already provided by drakvuf bundle:
https://github.com/tklengyel/drakvuf/blob/307420439af2f4caf92bcf80199d39ce22f2171e/package/postinst#L54

the user would get huge warning after installation of `drakvuf-bundle` DEB that the processor doesn't support EPT.

Reply:
hm but adding it to the docs is also pretty ok, I've added a note about checking for EPT support